device/send.go

```diff
@@ -306,6 +308,8 @@ top:
 		return
 	}
 
+	allZeros := [MaxMessageSize]byte{}
```

Comment:
If you declare this `const`, the compiler may be able to manage its life time better.

Reply:
It seems like arrays cannot be constant in Go unfortunately https://stackoverflow.com/questions/13137463/declare-a-constant-array